lib/node_modules/@stdlib/stats/mskmax/benchmark/benchmark.assign.js

```diff
@@ -0,0 +1,121 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var uniform = require( '@stdlib/random/array/uniform' );
+var zeros = require( '@stdlib/array/zeros' );
+var ndarray = require( '@stdlib/ndarray/base/ctor' );
+var pkg = require( './../package.json' ).name;
+var mskmax = require( './../lib' );
+
+
+// VARIABLES //
+
+var options = {
+	'dtype': 'float64'
+};
+
+
+// FUNCTIONS //
+
+/**
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} len - array length
+* @returns {Function} benchmark function
+*/
+function createBenchmark( len ) {
+	var mask;
+	var mbuf;
+	var out;
+	var x;
+	var i;
+
+	x = uniform( len, -50.0, 50.0, options );
+	x = new ndarray( options.dtype, x, [ len ], [ 1 ], 0, 'row-major' );
+
+	// Create a mask (mask out every 5th element):
+	mbuf = [];
+	for ( i = 0; i < len; i++ ) {
+		mbuf.push( ( i % 5 === 0 ) ? 1 : 0 );
+	}
```

Comment:
Use `random/array/bernoulli` and set the return dtype to `uint8`. No need for a manual loop here.